### PR TITLE
NOISSUE - Attest validation and verification

### PR DESCRIPTION
**Pull request by SammyOina**
to be merged after #66

### CODE REVIEW

File: cli/attestation.go [hunk 1]
```diff
@@ -3,22 +3,59 @@
...
 
 func (cli *CLI) NewAttestationCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:   "attestation",
-		Short: "Retrieve attestation information",
+		Use:   "attestation [command]",
+		Short: "Get and validate attestations",
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Println("Checking attestation")
+			fmt.Printf("Get and validate attestations\n\n")
```
Comment (WashingtonKK) [inline cli/attestation.go:24]: Can we use log.Println to be consistent?

Comment (WashingtonKK) [reply cli/attestation.go:24]: Use it in combination with fmt.Sprintf

Comment (SammyOina) [reply cli/attestation.go:24]: these are not log statements

**Review by WashingtonKK — APPROVED** [review]
LGTM

File: cmd/cli/main.go [hunk 2]
```diff
@@ -162,7 +162,12 @@ func main() {
 	agentCmd.AddCommand(cliSVC.NewAlgorithmsCmd())
 	agentCmd.AddCommand(cliSVC.NewDatasetsCmd())
 	agentCmd.AddCommand(cliSVC.NewResultsCmd())
-	agentCmd.AddCommand(cliSVC.NewAttestationCmd())
+	attestaionCmd := cliSVC.NewAttestationCmd()
+	agentCmd.AddCommand(attestaionCmd)
+
+	// attestation commands.
```
Comment (drasko) [inline cmd/cli/main.go:168]: Capitalize comment and remove full-stop at the end

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**